presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentOptions.java

```diff
@@ -29,16 +29,14 @@
     @Option(name = "--server-package", title = "server-package", description = "path to Presto server package")
     public File serverPackage = new File("presto-server/target/presto-server-${project.version}.tar.gz");
 
-    @Option(name = "--environment", title = "environment", description = "the name of the environment to start", required = true)
-    public String environment;
+    @Option(name = "--environment", title = "environment", description = "the name of the environment to start")
+    public String environment = "singlenode";
```

Comment:
revert?

Reply:
I don't know how to make environment optional (it's injected and then verified in SelectedEnvironmentProvider)